libr/include/r_util/r_graph_drawable.h

```diff
@@ -0,0 +1,53 @@
+#ifndef R2_GRAPH_DRAWABLE_H
+#define R2_GRAPH_DRAWABLE_H
+
+#include <r_types.h>
+#include <r_util/r_graph.h>
+#include <r_config.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Generic drawable graph node.
+ * 
+ * Provides minimal information to draw something without output format specific details.
+ */
+typedef struct r_anal_graph_node_info_t {
+	char *title;
+	char *body;
+	/**
+	 * @brief Optional offset for the object corresponding to node.
+	 * 
+	 * Interactive output modes can use it to provide actions like seeking to
+	 * this position or modify the object.
+	 */
+	ut64 offset;
+} RGraphNodeInfo;
+
+R_API void r_graph_free_node_info(void *ptr);
+R_API RGraphNodeInfo *r_graph_create_node_info(const char *title, const char *body, ut64 offset);
```

Comment:
I guess we can start by hiding this from the API as it's not used by any module outside the r_graph_add_node_info function.